content/docs/intro/concepts/stack.md

```diff
@@ -102,9 +106,11 @@ Current stack outputs (2):
 Use `pulumi stack select` to change stack; `pulumi stack ls` lists known ones
 ```
 
-## View stack outputs {#outputs}
+## Stack Outputs {#outputs}
+
+A stack can export values as stack outputs. These outputs are shown during an update, can be easily retrieved with the Pulumi CLI, and are displayed in the Pulumi Console. They can be used for important values like resource IDs and computed IP addresses and DNS names.
 
-When you use top-level exports in your Pulumi [program]({{< relref "programming-model#programs" >}}), they become [stack outputs]({{< relref "programming-model#stack-outputs" >}}). Stack outputs can be viewed via `pulumi stack output` and are shown on the stack information page on pulumi.com.
+Stack outputs can be viewed via `pulumi stack output` and are shown on the stack information page on app.pulumi.com.
 
 ### **JavaScript code**
```

Comment:
Now that this section includes the details of how to use stack outputs in code, I don't think we want a "JavaScript code" section here.  And indeed we probably want to clean up this section overall now that it's all in one place.

Reply:
Yup, agreed. I replaced what we had in this topic with the stacks output content from the programming model topic as it both had all of the supported languages and a bit more detail. PTAL.